lib/services/notify.rb

```diff
@@ -0,0 +1,37 @@
+require 'notifications/client'
+
+class Notify
```

Comment:
We don't need this class anymore, as all the Notify grunt work is handled by `mail-notify` and the mailers